### PR TITLE
made changes according to review

### PR DESCRIPTION
**Pull request by aditya-369**
hi @sushain97 sorry my naming to id's made delay in resolving this issue.Hope this push would be better naming.

### CODE REVIEW

File: assets/css/general.css [hunk 1]
```diff
@@ -26,6 +26,116 @@ h2 {
...
+    margin-right: .5em;
+    margin-top: 0;
+}
+
+#infoModalApertiumLogo {
+    height: 5em;
+    max-width: 100%;
+}
+
+#morphAnalyzerOutput {
```
Comment (sushain97) [inline assets/css/general.css]: As I said on the previous PR, this belongs in https://github.com/apertium/apertium-html-tools/blob/master/assets/css/analysis.css. Please do not open new PRs each time.

File: assets/css/general.css [hunk 2]
```diff
@@ -26,6 +26,116 @@ h2 {
...
+    border-spacing: 0;
+}
+
+#secondaryLocalGlobe {
```
Comment (sushain97) [inline assets/css/general.css]: "locale", not "local"

File: assets/css/general.css (hunk 1 above)
Comment (aditya-369) [reply assets/css/general.css]: Sorry @sushain97 i am new to git so i will try pushing to this branch and update this branch always and also regarding file upload and progress bar i will build and check and let you know if something goes wrong.I am bit slow as my exams are going on so sorry for delay.

**Comment by sushain97** [comment]
It looks fine to me on the Circle artifact.

**Comment by aditya-369** [comment]
so i can push the changes right?

File: .htmlhintrc.json [hunk 3]
```diff
@@ -10,7 +10,7 @@
   "alt-require": true,
   "doctype-html5": true,
   "style-disabled": true,
-  "inline-script-disabled": true,
+  "inline-script-disabled": false,
```
Comment (sushain97) [inline .htmlhintrc.json]: Not inline script, inline styles.

Comment (aditya-369) [reply .htmlhintrc.json]: okay sorry i will change update.

**Comment by aditya-369** [comment]
hi @sushain97  hope done all the changes required.Thanks

**Comment by aditya-369** [comment]
hi @sushain97 ,made the htmlhint `style-disabled` false and also verified by  building and  it works fine unlike before snapshot.

**Comment by aditya-369** [comment]
hi @sushain97 done with consistency changes.Sorry for delay.I was also looking at #73 detection so any suggestion where to start?And also for gsoc i was late for submitting so did not review with u.Hope its fine.

File: assets/css/general.css [hunk 4]
```diff
@@ -26,6 +26,112 @@ h2 {
...
+.font-weight-bold {
+    font-weight: bold;
+}
+/* sass-lint:enable class-name-format */
+
+#apertiumLogo {
+    margin-bottom: -.25em;
+}
+
+#gsocRef,
```
Comment (sushain97) [inline assets/css/general.css]: The "ref" are still here?

File: index.html.in [hunk 5]
```diff
@@ -402,28 +402,28 @@
...
                         <div data-text="What_Is_Apertium"></div>
-                        <div data-text="Maintainer" style="padding-bottom: 2em;"></div>
+                        <div data-text="Maintainer" id="maintainerAbout"></div>
 
                         <div class="row lead">
                             <div class="col-md-6">
                                 <div class="center-block">
-                                    <a href="https://developers.google.com/open-source/soc/" target="_blank" style="text-decoration: none" rel="noopener">
-                                        <img alt="Google Summer of Code" data-src="https://summerofcode.withgoogle.com/static/img/summer-of-code-logo.svg" style="height: 2.5em;">
+                                    <a href="https://developers.google.com/open-source/soc/" target="_blank" id="gsocRef" rel="noopener">
```
[the remark this answers is not in the document]
Comment (sushain97) [reply index.html.in]: This ^

File: index.html.in [hunk 6]
```diff
@@ -402,28 +402,28 @@
                     </div>
                     <div class="modal-body">
                         <div data-text="What_Is_Apertium"></div>
-                        <div data-text="Maintainer" style="padding-bottom: 2em;"></div>
-
+                        <div data-text="Maintainer" id="aboutModalMaintainer"></div>
+
```
Comment (sushain97) [inline index.html.in]: Trailing whitespace?

Comment (aditya-369) [reply index.html.in]: something went wrong in syncing in vscode sorry.I will re-push it.

File: index.html.in [hunk 7]
```diff
@@ -285,14 +285,14 @@
                               </div>
                           </div>
                           <div class="form-group">
-                            <div class="col-sm-offset-2 col-sm-10 col-md-offset-1">
```
[the remark this answers is not in the document]
Comment (aditya-369) [reply index.html.in]: sorry @sushain97, in syncing between css and index.html.in i over looked.

[the remark this answers is not in the document]
Comment (sushain97) [reply index.html.in]: Looks like there's an extra space now.

**Comment by aditya-369** [comment]
@sushain97 made changes in index.html.in which were not consistent.Hope so done and ready to review my mistakes.

**Comment by sushain97** [comment]
See my previous comment.

File: index.html.in [hunk 8]
```diff
@@ -369,15 +369,15 @@
         <div id="footer">
             <div class="navbar-bottom hidden-xs">
                 <div class="container" id="footer-container">
-                    <ul class="nav nav-pills pull-left" style="display: inline-block;">
+                    <ul class="nav nav-pills pull-left d-inline-block ">
```
Comment (aditya-369) [inline index.html.in]: @sushain97 ,this was the extra space you were refering to right?

Comment (sushain97) [reply index.html.in]: No.

File: index.html.in [hunk 9]
```diff
@@ -318,7 +318,7 @@
                               </div>
                           </div>
                           <div class="form-group">
-                            <div class="col-sm-offset-2 col-sm-10 col-md-offset-1">
+                            <div class="col-sm-offset-2 col-sm-10  col-md-offset-1">
```
Comment (sushain97) [inline index.html.in]: There is an extra space here.

Comment (aditya-369) [reply index.html.in]: sorry sleepy head mistake. :D

**Comment by aditya-369** [comment]
Thanks @sushain97 sorry if i disturbed u for many times and thanks for the patience as i know how noob i was. Regard to my next PR i want to start with #73 so starting from existing work of @Kira-D work and improving it or are there any options to start it.

**Comment by sushain97** [comment]
No worries, @aditya-369. Everyone starts somewhere. The language detection work is more on the APy side than the HTML-tools side.

**Comment by sushain97** [comment]
(@ftyers also knows more about what Kira did wrt language detection than I do)

**Pull request merged**